### PR TITLE
updated the mentors page

### PR DESCRIPTION
**Pull request by Illaf**
## Related Issue

Closes: #[issue number]

## Checklist:

- [ ] My code adheres to the established style guidelines of this project.
- [ ] I have conducted a self-review of my code.
- [ ] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [ ] My changes have not introduced any new warnings.

## Screenshots

|           Original           |           Updated           |
| :--------------------------: | :-------------------------: |
| Original screenshot | Updated screenshot |

Please provide any necessary screenshots to illustrate the changes made.

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -1645,3 +1645,53 @@ select {
...
+    padding-bottom: 42px;
+    border-bottom: 2px solid grey;
+   margin-left: 12px;
+   margin-right: 12px;
+}
+.mentor-div2{
+  justify-content: center;
+  top: 0px;
+  margin-top: 52px;
+  margin-left: 104px;
```
Comment (anmode) [inline styles/globals.css:1672]: move it to mentorProfile css file.. as it is not global property!

**Review by anmode — COMMENTED** [review]
looks cool

File: styles/globals.css (hunk 1 above)
Comment (Illaf) [reply styles/globals.css]: Done👍

**Comment by Illaf** [comment]
@anmode updated the testimonials section . Please check.

**Comment by anmode** [comment]
cool,
1. Remove the grey background.. simple theek h
2. Needs to be responsive in mobile mode

**Comment by Illaf** [comment]
@anmode I have added the changes. One thing I wanted to mention is that I have commented out the links because they were giving errors(not defined) @sayyedarib If you could check it out why. Also I think book session is not working. Please check.

**Comment by anmode** [comment]
Can you create a component folder named newMentorProfile?
and put all component in it, mentorAbout, sessionCard , testimonialCard ??

**Comment by Illaf** [comment]
@anmode The pr has following glitches please check :
The font size of sessions heading is not changing  I even marked it important.
The font size of testimonials content is not changing

**Comment by anmode** [comment]
```


Failed to compile.
--
23:27:00.582 |  
23:27:00.582 | ./hook/useAPi.js:68:2
23:27:00.582 | Syntax error: Unexpected token
23:27:00.582 |  
23:27:00.583 | 66 \|   //   return apidata;
23:27:00.583 | 67 \|   // }
23:27:00.583 | > 68 \|
23:27:00.584 | \|   ^

```

File: hook/useAPi.js
Comment (anmode) [inline hook/useAPi.js:1]: why you changed this file?? it is not related?

File: components/newMentorProfile/sessions.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
+import React,{ useState } from 'react'
+import SessionCard from "../../components/mentorProfile/components/SessionCard";
+import styles1 from "../../styles/mentorTestimonial.module.css";
+function Sessions({mentorDetail}) {
+  const [modalPopup, setModalPopup] = useState(false);
```
Comment (anmode) [inline components/newMentorProfile/sessions.js:5]: I think this is not the component, the Session card is the component. 
May we can have this is `[mentorUsername].page.js`as before.

File: components/newMentorProfile/sessions.js [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        sessionMeetingDuration,
+        priceSession
+      );
+    } else {
+      const redirectUrl = window.location.href;
+      sessionStorage.setItem("redirectUrl", redirectUrl);
+      router.push(`/userAuth#login`);
+    }
+  };
+  const handleBookSession = async (
```
Comment (anmode) [inline components/newMentorProfile/sessions.js:50]: this whole functionality is part of [mentorusername].page.js...I wonder if it is working?

File: pages/[mentorUsername].page.js [hunk 4]
```diff
@@ -107,58 +113,34 @@ function Index({ mentorDetail }) {
...
-              {mentorDetail?.bookSession?.length !== 0 &&
-                mentorDetail?.bookSession?.map((session, index) => (
-                  <SessionCard
-                    key={index}
-                    type={session?.sessionType}
-                    name={session?.sessionName}
-                    description={session?.sessionDescription}
-                    duration={session?.sessionMeetingDuration}
-                    pricePerSession={session?.priceSession}
-                    handleBookSession={() => {
```
Comment (anmode) [inline pages/[mentorUsername].page.js:138]: as you created new component so, like wise before it is good to import component here and not to remove the funtionality of sessionBooking
@sayyedarib  or we can have a particular booking Session component for it???

**Review by anmode — CHANGES_REQUESTED** [review]
@Illaf good,
Left some comment this PR needs to be addressed

File: hook/useAPi.js
Comment (Illaf) [reply hook/useAPi.js:1]: Initially it was giving errors and then had  merge conflicts

File: components/newMentorProfile/sessions.js (hunk 2 above)
Comment (Illaf) [reply components/newMentorProfile/sessions.js:5]: The session card component is imported inside the sessions.js ,which is then placed in mentor username as a component.

File: components/newMentorProfile/sessions.js (hunk 3 above)
Comment (Illaf) [reply components/newMentorProfile/sessions.js:50]: yes it is. I'll remove it from sessions.js

**Comment by anmode** [comment]
Cool, Ilaf!! Good

**Pull request merged**